lessons/en/chapter_5.yaml

```diff
@@ -29,32 +29,49 @@
   code: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=struct%20Foo%20%7B%0A%20%20%20%20x%3A%20i32%2C%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20let%20foo_a%20%3D%20Foo%20%7B%20x%3A%2042%20%7D%3B%0A%20%20%20%20let%20foo_b%20%3D%20Foo%20%7B%20x%3A%2013%20%7D%3B%0A%0A%20%20%20%20println!(%22%7B%7D%22%2C%20foo_a.x)%3B%0A%0A%20%20%20%20println!(%22%7B%7D%22%2C%20foo_b.x)%3B%0A%20%20%20%20%2F%2F%20foo_b%20is%20dropped%20here%20%0A%20%20%20%20%2F%2F%20foo_a%20is%20dropped%20here%0A%7D%0A
 - title: Dropping is Hierarchical
   content_markdown: >
-    When a struct is dropped, the struct itself is dropped first, then its
-    children are dropped individually, and so on.
+    When a struct is dropped, its fields (children) are dropped first, in the 
```

Comment:
This is incorrect, take a look at the [`Drop`](https://doc.rust-lang.org/std/ops/trait.Drop.html) trait.